.github/workflows/test.yml

```diff
@@ -145,6 +145,9 @@ jobs:
           sudo cpanm -n Neovim::Ext || cat "$HOME/.cpanm/build.log"
           perl -W -e 'use Neovim::Ext; print $Neovim::Ext::VERSION'
 
+      - name: Remove .git directory
+        run: cmake -E rm -rf -- .git
```

Comment:
This is an expensive fix for a problem that probably won't happen again unless we create another root traversal function. I think a better tradeoff would just be to fix the tests and call it a day.

Reply:
I'm going to fix the tests. This just helps ensure there aren't other things relying on git (like the build part of the version inf). I'm not sure what's expensive about it. It takes less than a second in CI.

Reply:
I meant more the maintaining this is expensive compared to the problem, which probably won't occur again.